#! /usr/bin/python3

# This will my first attempt to force exceptions
# capture them and
# deal with them

